addon/components/cf-field/input/file.js

```diff
@@ -0,0 +1,64 @@
+import Component from "@ember/component";
+import { inject as service } from "@ember/service";
+import { reads } from "@ember/object/computed";
+
+import getFileAnswerInfoQuery from "ember-caluma/gql/queries/get-fileanswer-info";
+import layout from "../../../templates/components/cf-field/input/file";
+import { computed } from "@ember/object";
+
+export default Component.extend({
+  layout,
+  tagName: "",
+
+  intl: service(),
+  apollo: service(),
+
+  downloadUrl: reads("field.answer.fileValue.downloadUrl"),
+
+  placeholder: computed("field.answer", function() {
+    return this.get("field.answer.fileValue")
+      ? this.intl.t("caluma.form.changeFile")
+      : this.intl.t("caluma.form.selectFile");
+  }),
+
+  actions: {
+    async download() {
+      const { downloadUrl } = await this.apollo.watchQuery(
+        {
+          query: getFileAnswerInfoQuery,
+          variables: {
+            id: this.field.answer.id
+          },
+          fetchPolicy: "cache-and-network"
+        },
+        "node.fileValue"
+      );
+
+      if (downloadUrl) {
+        window.open(downloadUrl, "_blank");
+      }
+    },
+
+    async save({ target }) {
+      const file = target.files[0];
+
+      if (!file) {
+        return;
+      }
+
+      const saved = await this.onSave(file.name);
+
+      const data = new FormData();
```

Comment:
Just to be sure: This works in IE11, right? ([caniuse](https://caniuse.com/#feat=xhr2) just mentions limited support for JSON content types)

Reply:
This shouldn't be a problem as we append a file. But I'll check anyway.

Reply:
I haven't gotten to test the FormData. IE 11 breaks long before that on a `class` statement and an arrow function. Probably needs some modification to the build process to transform the things IE doesn't understand. But I suggest we open another ticket for that as there are many more issues with IE. (i.e. I remember seeing errors about `Object` missing `assign` from a previous test.

Reply:
Oh, good to know! This sounds like something we should look at with high priority then.